addons/mail/static/src/components/chatter/chatter.xml

```diff
@@ -4,46 +4,50 @@
     <t t-name="mail.Chatter" owl="1">
         <div class="o_Chatter">
             <t t-if="chatter">
-                <ChatterTopbar
-                    class="o_Chatter_topbar"
-                    chatterLocalId="chatter.localId"
-                />
-                <t t-if="chatter.threadView and chatter.isComposerVisible">
-                    <Composer
-                        class="o_Chatter_composer"
-                        t-att-class="{ 'o-bordered': chatter.hasExternalBorder }"
-                        composerLocalId="chatter.thread.composer.localId"
-                        hasFollowers="true"
-                        hasMentionSuggestionsBelowPosition="true"
-                        isCompact="false"
-                        isExpandable="true"
-                        textInputSendShortcuts="['ctrl-enter', 'meta-enter']"
-                        t-on-o-message-posted="_onComposerMessagePosted"
-                        t-ref="composer"
-                    />
-                </t>
-                <t t-if="chatter.isAttachmentBoxVisible">
-                    <AttachmentBox
-                        class="o_Chatter_attachmentBox"
-                        threadLocalId="chatter.thread.localId"
-                    />
-                </t>
-                <t t-if="chatter.thread and chatter.hasActivities and chatter.thread.activities.length > 0">
-                    <ActivityBox
-                        class="o_Chatter_activityBox"
+                <div class=".o_Chatter_fixedPanel">
```

Comment:
Remove . in name, also it's not really a "fixed" panel, it's just a normal panel. Feels like "composer" should actually be part of "chattertopbar" and the "fixed panel" is the top bar itself. But that's out of scope.

Reply:
do you prefer .o_Chatter_topbarPanel or topPanel or something like that ?

Reply:
I don't have a better idea now. Technically speaking it is `o_Chatter_topbarAndComposer`, but we might want a more functional name to describe this area. How would we call it when talking about it? I think the "fixed area" makes sense in functional terms actually, even if it's not technically fixed.

Reply:
What about `.functionallyFixedButNotTechnicallyPanel` ?